code-examples/quarkus/kafka-quickstart/src/main/resources/application.properties

```diff
@@ -16,13 +16,31 @@ quarkus.container-image.builder=docker
 quarkus.kubernetes.deployment-target=kubernetes
 quarkus.container-image.push=true
 
-# Configure the kafka broker to use the service
-kafka.bootstrap.servers=${BOOTSTRAP_SERVER}
-kafka.security.protocol=SASL_SSL
+## Service binding configuration used by default
 
-kafka.sasl.mechanism=OAUTHBEARER
+quarkus.kubernetes-service-binding.root=/bindings
```

Comment:
@pmuir What you think about profiles approach?

Reply:
I like how we `killed couple birds by one stone`. 

If this sounds complex we can simplify:
- assume that env vars are used only in dev profile
- assume that sbo are used in production
- remove sbo env (although it gives value as it list names of the env variables that people might use outside quarkus as example

Reply:
* we can drop the dev-env-sbo profile I think
* if we enable SBO here like this, what's the impact on some who isn't using SBO?

Reply:
I think it's probably better to use a profile for SBO as well, this way it's explicit for the user to enable it.